README.md

```diff
@@ -1,15 +1,56 @@
-# VirtualCam Avatar
+<!-- omit in toc -->
+# Avatar Animator
```

Comment:
Could you also change the app title in the html file to the new name?

Reply:
done in cda6cff